boards/mulle/include/board.h

```diff
@@ -104,8 +102,10 @@ void board_init(void);
 
 /**
  * @name Define the interface to the AT86RF212B radio
- *
- * @todo Work around missing RESET pin on Mulle v0.6x
+ * @{
+ */
```

Comment:
Don't need to close the doxygen block here

Reply:
I will try again, but I strange error on `make doc` without splitting this block. I will paste the results here

Reply:
this is my output when I don't split the block:

```
( cat riot.doxyfile ; echo "GENERATE_HTML = yes" ) | doxygen -
todo:17: warning: Unexpected character `b'
todo:17: warning: Unexpected character `o'
todo:17: warning: Unexpected character `a'
todo:17: warning: Unexpected character `r'
todo:17: warning: Unexpected character `d'
todo:17: warning: Unexpected character `s'
todo:17: warning: Unexpected character `/'
todo:17: warning: Unexpected character `m'
todo:17: warning: Unexpected character `u'
todo:17: warning: Unexpected character `l'
todo:17: warning: Unexpected character `l'
todo:17: warning: Unexpected character `e'
todo:17: warning: Unexpected character `/'
todo:17: warning: Unexpected character `i'
todo:17: warning: Unexpected character `n'
todo:17: warning: Unexpected character `c'
todo:17: warning: Unexpected character `l'
todo:17: warning: Unexpected character `u'
todo:17: warning: Unexpected character `d'
todo:17: warning: Unexpected character `e'
todo:17: warning: Unexpected character `/'
sh: latex: no se encontró la orden
error: Problems running latex. Check your installation or look for typos in _formulas.tex and check _formulas.log!
sh: dvips: no se encontró la orden
error: Problems running dvips. Check your installation!
make[1]: se sale del directorio '/home/jialamos/Development/RIOT/doc/doxygen
```

Maybe a Doxygen bug?

Reply:
What about moving the `@todo` before the `@{` ?

Reply:
same results. Does it work for you? Maybe my Doxygen version has a bug

Reply:
You can drop the todo. It's not important

Reply:
I also have the issue, but it works with the actual version of your branch. So let's keep it like this

Reply:
or maybe I just drop the `@todo` as @gebart proposed, so the change is minimal